packages/scandipwa/src/route/MyAccount/MyAccount.style.scss

```diff
@@ -41,17 +41,27 @@
 
         &_activeTab {
             &_my-orders {
-                .MyAccount-SubHeading {
-                    border-radius: 3px;
-                    border: var(--my-account-order-subheading-border);
-                    bottom: 4px;
-                    color: var(--my-account-order-subheading-color);
-                    font-size: 14px;
-                    font-weight: 400;
-                    margin-inline-start: 10px;
-                    margin: 3px 0 0;
-                    padding: 9px 10px 5px;
-                    text-transform: uppercase;
+                .MyAccount {
+                    &-Heading {
+                        @include desktop {
+                            display: flex;
+                        }
+                    }
+
+                    &-SubHeading {
+                        display: block;
+                        border-radius: 3px;
+                        border: var(--my-account-order-subheading-border);
+                        bottom: 4px;
+                        color: var(--my-account-order-subheading-color);
+                        font-size: 14px;
+                        font-weight: 400;
+                        margin-inline-start: 10px;
+                        margin: 3px 0 0;
```

Comment:
This makes no sense since you override margin-inline-start (margin-left for LTR) with margin on the next line.
Also when you put three values, you actually put different top and bottom. Please don't use three values, use logical CSS properties instead.